src/utilities/Error.js

```diff
@@ -13,16 +13,15 @@ class Error {
  */
   static handleErrorResponse(res, error) {
     if (error.errors) {
-      res.status(400).json({
+      return res.status(400).json({
         success: false,
         message: error.errors[0].message
       });
-    } else {
-      res.status(500).json({
-        success: false,
-        message: 'Internal server error',
-      });
     }
+    return res.status(500).json({
```

Comment:
also, you should consider importing the helper function adopted by the team for the `Internal server error`
I suggest you replace 
```
 res.status(500).json({
        success: false,
        message: 'Internal server error',
      });
```
with
   ```
errorResponse.handleErrorResponse(res, error);
```


```suggestion
    errorResponse.handleErrorResponse(res, error);
```